ui/src/containers/TaskContainer.js

```diff
@@ -289,9 +265,6 @@ function mapDispatchToProps(dispatch) {
     ),
     updateTask: bindActionCreators(updateTask, dispatch),
     addTask: bindActionCreators(addTask, dispatch),
-    addSamplesToList: bindActionCreators(addSamplesToList, dispatch),
-    addSamplesToQueue: bindActionCreators(addSamplesToQueue, dispatch),
-    addSampleAndMount: bindActionCreators(addSampleAndMount, dispatch),
```

Comment:
The `AddSample` component is already wrapped with a container connected to the Redux store, so we can move these bindings down to that container and remove props `addToQueue` and `addAndMount`.